joint_trajectory_controller/src/joint_trajectory_controller_parameters.yaml

```diff
@@ -101,6 +101,11 @@ joint_trajectory_controller:
         default_value: 0.0,
         description: "Feed-forward scaling of velocity."
       }
+      normalize_error: {
```

Comment:
I am even open to have this here. It is not fully off - it is quite related to the PID stuff in JTC...